lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -15,67 +13,42 @@ class FeatureCard extends React.Component {
   constructor(props) {
     super(props);
 
-    // These variables shouldn't force a state change & subsequent render
-    this.requests = [];
-    this.campaigns = [];
-    this.campaignsToExclude = setting('dsUser.activity.signups', []);
-    this.campaignsApiPage = 1;
-    this.apiClient = new ApiClient('v1');
-
     this.state = {
       actionDisabled: false,
       campaignIndex: -1,
       isSignedUp: false,
       isLoading: false,
     };
 
+    this.campaigns = [];
+    this.controller = new GalleryController();
```

Comment:
It does feel strange that each card needs it's own GalleryController, since that means they'll need to make three independent HTTP requests too. I don't want to over-engineer this, but I think it'd make more sense to inject the same "controller" into each feature card so that they can all shared the same API request & data source.